dnf-behave-tests/features/steps/common/file.py

```diff
@@ -6,6 +6,41 @@
 import codecs
 import os
 import shutil
+import re
+
+import gzip
+import bz2
+
+# xz compression
+try:
+    import lzma
+except ImportError:
+    from backports import lzma
+
+def regex_find_file(filepath):
+    files = os.listdir(os.path.dirname(filepath))
+    filepath = filepath + '$'
+    pattern = re.compile(os.path.basename(filepath))
+    matched_files = list(filter(pattern.match, files))
+    match_count = len(matched_files)
+    if (match_count > 1):
+        raise AssertionError("Multiple files matching: {0}, found:\n{1}" .format(filepath, '\n'.join(matched_files)))
```

Comment:
You are printing filepath with '$' in the end.
You probably should not change filepath and do something like
pattern = re.compile(os.path.basename(filepath) + '$')